README.md

```diff
@@ -41,119 +38,10 @@ If you’re using this app, please ★Star the repository to show your interest!
-
-4. **Get credentials for the created cluster**
-
-- GKE Autopilot mode:
-
-```
-gcloud container clusters get-credentials bank-of-sirius \
-  --project=${PROJECT_ID} --region=${REGION}
-```
-
-- GKE Standard mode:
-
-```
-gcloud container clusters get-credentials bank-of-sirius \
-  --project=${PROJECT_ID} --zone=${ZONE}
-```
-
-5. **Deploy the demo JWT public key** to the cluster as a Secret. This key is used for user account creation and authentication.
-
-```
-kubectl apply -f ./extras/jwt/jwt-secret.yaml
-```
-
-6. **Deploy the sample app to the cluster.**
-
-```
-kubectl apply -f ./kubernetes-manifests
-```
-
-7. **Wait for the Pods to be ready.**
-
-```
-kubectl get pods
-```
-
-After a few minutes, you should see:
-
-```
-NAME                                  READY   STATUS    RESTARTS   AGE
-accounts-db-6f589464bc-6r7b7          1/1     Running   0          99s
-balancereader-797bf6d7c5-8xvp6        1/1     Running   0          99s
-contacts-769c4fb556-25pg2             1/1     Running   0          98s
-frontend-7c96b54f6b-zkdbz             1/1     Running   0          98s
-ledger-db-5b78474d4f-p6xcb            1/1     Running   0          98s
-ledgerwriter-84bf44b95d-65mqf         1/1     Running   0          97s
-loadgenerator-559667b6ff-4zsvb        1/1     Running   0          97s
-transactionhistory-5569754896-z94cn   1/1     Running   0          97s
-userservice-78dc876bff-pdhtl          1/1     Running   0          96s
-```
-
-8. **Access the web frontend in a browser** using the frontend's `EXTERNAL_IP`.
-
-```
-kubectl get service frontend | awk '{print $4}'
-```
-
-*Example output - do not copy*
-
-```
-EXTERNAL-IP
-35.223.69.29
-```
 
 ## Other Deployment Options
 
-- **Workload Identity**: [See these instructions.](./docs/workload-identity.md)
-- **Cloud SQL**: [See these instructions](./extras/cloudsql) to replace the in-cluster databases with hosted Google Cloud SQL.
-- **Multicluster with Cloud SQL**: [See these instructions](./extras/cloudsql-multicluster) to replicate the app across two regions using GKE, Multi-cluster Ingress, and Google Cloud SQL.
-- **Istio**: Apply `istio-manifests/` to your cluster to access the frontend through the IngressGateway.
-- **Sirius Service Mesh**: ASM requires Workload Identity to be enabled in your GKE cluster. [See the workload identity instructions](./docs/workload-identity.md) to configure and deploy the app. Then, apply `istio-manifests/` to your cluster to configure frontend ingress.
-- **Java Monolith (VM)**: We provide a version of this app where the three Java microservices are coupled together into one monolithic service, which you can deploy inside a VM (eg. Google Compute Engine). See the [ledgermonolith](./src/ledgermonolith) directory.
+- **Java Monolith (VM)**: We provide a version of this app where the three Java microservices are coupled together into one monolithic service, which you can deploy inside a VM. See the [ledgermonolith](./src/ledgermonolith) directory.
```

Comment:
We deleted the monolith from this repo. Perhaps we want to strike this section.

Reply:
Ah, forgot. Removed.